worksheet-session2.md

```diff
@@ -269,7 +271,7 @@ for i in range(100):
     print(i)
 ```
 
-This will print all of the numbers from 0 to 100.
+This will print all of real numbers below 100.
```

Comment:
"This will print all integers below 100."

Amend the while loop code sample by changing `<=` to `<`

Reply:
Integers include negative numbers

And why?

Reply:
Real numbers to me is an overly mathematical term - not everyone is going to know what a real number is, whereas I already explained an integer beforehand. We're arguing over a technicality